packages/connect-node/src/private/validate-read-max-bytes-option.ts

```diff
@@ -16,6 +16,9 @@ import { Code, ConnectError } from "@bufbuild/connect-core";
 
 const maxReadMaxBytes = 0xffffffff; // zlib caps maxOutputLength at this value
 
+/**
+ * @deprecated use compressionValidateOptions from @bufbuild/connect-core
+ */
```

Comment:
Deprecated a couple of functions that are still used in transports. They will be replaced when we switch the Node transports to use asynchronous iterables.